resources/scripts/components/dashboard/DashboardContainer.tsx

```diff
@@ -58,8 +81,14 @@ export default () => {
                     <Switch
                         name={'show_all_servers'}
                         defaultChecked={showOnlyAdmin}
-                        onChange={() => setShowOnlyAdmin(s => !s)}
+                        onChange={() => setShowOnlyAdmin((s) => !s)}
                     />
+                    <div css={tw`mx-3`}>
```

Comment:
Shouldn't it be `ml-3`? How does it currently look?

Reply:
Looks fine.

Reply:
Need a screenshot. Because the switch doesn't have a margin right (without your PR) and you added it to your element